src/pages/tBTC/Bridge/Minting/MakeDeposit.tsx

```diff
@@ -40,75 +49,64 @@ const AddressRow: FC<
 
   return (
-    <>
+    <VStack spacing="2" align="start" {...restProps}>
       <HStack
         alignItems="center"
-        mb="3.5"
         // To center the tooltip icon. The tooltip icon is wrapped by `span`
         // because of: If you're wrapping an icon from `react-icons`, you need
         // to also wrap the icon in a `span` element as `react-icons` icons do
         // not use forwardRef. See
         // https://chakra-ui.com/docs/components/tooltip#with-an-icon.
         sx={{ ">span": { display: "flex" } }}
       >
-        <BodyMd color={titleColor}>BTC Deposit Address</BodyMd>
-        <TooltipIcon
-          color={titleColor}
-          label="This is a unique BTC address generated based on the ETH address and Recovery address you provided. Send your BTC funds to this address in order to mint tBTC."
-        />
+        <BodySm fontWeight="medium" color="brand.100" lineHeight={1.5}>
+          BTC Deposit Address
+        </BodySm>
+        <TooltipIcon label="This is a unique BTC address generated based on the ETH address and Recovery address you provided. Send your BTC funds to this address in order to mint tBTC." />
       </HStack>
-      <BTCAddressCard p="2.5" display="flex" justifyContent="center">
-        <Box
-          p={2.5}
-          backgroundColor={"white"}
-          width={"100%"}
-          maxW="205px"
-          borderRadius="8px"
-        >
+      <Stack
+        w="full"
+        direction={{ base: "column", sm: "row" }}
+        align="start"
+        py={4}
+        pl={4}
+        pr={10}
+        spacing={10}
+        rounded="2xl"
+        border="1px solid #333"
+      >
+        <Box p={3.5} rounded="xl" bg="brand.100">
           <QRCode
-            size={256}
-            style={{ height: "auto", maxWidth: "100%", width: "100%" }}
+            size={180}
             value={btcDepositAddress}
-            viewBox={`0 0 256 256`}
+            viewBox="0 0 180 180"
+            bgColor="transparent"
           />
         </Box>
-      </BTCAddressCard>
-      <CopyToClipboard textToCopy={btcDepositAddress}>
-        <HStack mt="2.5">
-          <BTCAddressCard minW="0" p="2">
-            <BodyMd color={btcAddressColor}>{btcDepositAddress}</BodyMd>
-          </BTCAddressCard>
-          <BTCAddressCard
-            flex="1"
-            p="4"
-            display="flex"
-            alignSelf="stretch"
-            alignItems="center"
+        <VStack spacing={6} align="start">
+          <BodyMd color="brand.100" overflowWrap="anywhere">
+            {btcDepositAddress}
+          </BodyMd>
+          <Button
+            size="sm"
+            variant="outline"
+            onClick={handleCopy}
+            color="white"
+            textTransform="capitalize"
+            rightIcon={<Icon as={CopyIcon} color="brand.100" />}
           >
-            <CopyToClipboardButton />
-          </BTCAddressCard>
-        </HStack>
-      </CopyToClipboard>
-    </>
+            Copy
```

Comment:
Non-blocking but good to have:

Can we change the text to "Copied!" once we click `Copy` button?

I thinks it should be easy based on [the docs](https://v1.chakra-ui.com/docs/styled-system/utility-hooks/use-clipboard#usage). We should just use `hasCopied` from `useClipboard` hook.

I marked is as non-blocking, but would be nice to have since it's a quick fix and makes the user experience better.

Reply:
Ref commit: affa52babcf61b49245b05a9bb8ae32556659846